packages/@azure/eventhubs/client/changelog.md

```diff
@@ -1,3 +1,16 @@
+### 2019-03-26 2.0.0
+
+#### Breaking Changes
+- If you have been using the `createFromAadTokenCredentials` function to create an instance of the `EventHubClient`, you will now need to use the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) library instead of [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) library to create the credentials that are needed by the `createFromAadTokenCredentials` function.
```

Comment:
Would it make sense to show a small snippet of code for how to do that?

Reply:
Snippet added